samples/Patient and Population Services G10/docs/deployment.md

```diff
@@ -1,18 +1,21 @@
 # Sample Deployment: Azure ONC (g)(10) & SMART on FHIR Sample
 
-This document will guide you through the steps needed for deploying this sample. Since this sample is a combination of several Azure components, custom code, and Azure Active Directory configuration, it is not all automated and will require at least a couple of hours.
+This document guides you through the steps needed to deploy this sample. This sample deploys Azure components, custom code, and Azure Active Directory configuration.
+
+Note : This sample is not automated and on average will require at least a couple of hours to deploy end to end.
 
 ## 1. Prerequisites
 
 Before deploying this sample, you will need to install some Azure tools **and** ensure you have administrator access to an Azure subscription / tenant.
 
 - Make sure you have the pre-requisites applications installed on your computer.
-  - Azure Developer CLI: Please install this via [the instructions here](https://learn.microsoft.com/azure/developer/azure-developer-cli/install-azd?tabs=baremetal%2Cwindows)
+   - Azure Developer CLI: Please install this via [the instructions here](https://learn.microsoft.com/azure/developer/azure-developer-cli/install-azd?tabs=baremetal%2Cwindows)
   - Visual Studio or Visual Studio Code (for debugging the sample code).
   - NPM (for debugging sample).
-- .NET SDK installed (the version specified in [global.json](../../../global.json) (for building the sample).
-- Access to an Azure Subscription where you can create resources and add role assignments.
-- Elevated access in Azure Active Directory to create Application Registrations and assign Azure Active Directory roles.
+  - .NET SDK installed (Ensure to install the version specified in [global.json](https://github.com/microsoft/fhir-server/blob/main/global.json) (for building the sample).
```

Comment:
We don't have `global.json` anymore since the repo was moved. Can you remove this?

Reply:
Thanks and addressed